src/types.h

```diff
@@ -38,6 +39,7 @@ typedef struct {
  * stop info struct.
  */
 typedef struct {
+    char outname[33];
```

Comment:
this should also be 16

Reply:
@zvr isn't that too low? `ulimit -Hn` returns 524288 in my system. I am not against dynamic memory but this is easily solve-able with a few extra bytes of static memory.

Reply:
Even the whole Linux kernel has less than 100K files; even a `[8]` would be fine.
A `[16]` will probably cover a build of a whole distribution.

Reply:
@zvr Oh okay cool to know some statistics. I will make it 17 tho account for the 1 byte reserved for `[fp]` :)